.github/workflows/master_habbittrack.yml

```
@@ -0,0 +1,65 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)